week5.ipynb

```diff
@@ -0,0 +1,366 @@
+{
```

Comment:
<p>I like the lines separating questions! How'd you do that?</p><p> </p><p>On question 2, you say "Create a variable and list of values of your own..." - I think what you want them to create is a dictionary (which is different from a list in python).</p><p> </p><p>In question 4, the conversion from Kelvin to C could be formatted using LaTeX</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CUB-Computational-Tools/2020-ps-jolsenvaldez/pull/6/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>